cfgov/unprocessed/apps/find-a-housing-counselor/css/hud_print.css

```diff
@@ -54,136 +65,51 @@
         padding: 0;
     }
 
-    p,
-    h1,
-    h2,
-    h3,
-    h4 {
-        orphans: 3;
-        widows: 3;
-    }
-
-    h1,
-    h2,
-    h3,
-    h4 {
-        page-break-after: avoid;
-    }
-
-    /* Hide unneeded content */
-    .o-header,
-    .bread,
-    .share,
-    .o-footer,
-    .hud_hca_api_print_hide {
-        display: none !important;
-    }
-
-    /* Some adjustments */
-    table, thead, tbody, tfoot, tr, td, th, caption {
-        border: 0 !important;
-        display: block !important;
-        margin: 0 !important;
-        padding: 0 !important;
-    }
-
-    .pagetitle > h1 {
-        font-size: 16pt;
-    }
-
-    img.hud_hca_api_print_logo {
-        float: right;
-        max-width: 20% !important;
-    }
-
-    .cfpb_hud_hca_api_print_text {
-        border-bottom: 4px solid #101820;
-        padding-bottom: 1em;
-    }
-
-    .hud_hca_api_the_results {
-        font-weight: bold;
-        text-transform: uppercase;
-    }
-
-    /* Grid adjustments */
-    tr.hud_hca_api_result:before,
-    tr.hud_hca_api_result:after {
-        content: " ";
-        display: table;
-    }
-
-    tr.hud_hca_api_result:after {
-        clear: both;
-    }
-
-    tr.hud_hca_api_result {
-        *zoom: 1;
+    /* Prevent blank sheet at the end of the page. */
+    main,
+    .content_main {
+      margin-bottom: 0;
+      padding-bottom: 0;
     }
 
-    tr.hud_hca_api_result {
-        border-bottom: 1px solid #babbbd !important;
-        padding: 1.5em 0 !important;
+    /* Item result row. */
+    tr {
+      display: grid !important;
```

Comment:
What does this look like in a browser that doesn't support CSS Grid Layout?

Reply:
🤦‍♂️ Nevermind. I was on the wrong branch.

Reply:
This is IE9. Later IEs look equivalent.

<img width="447" alt="screen shot 2018-12-21 at 5 32 29 pm" src="https://user-images.githubusercontent.com/704760/50366237-82b10680-0546-11e9-812d-70bb0fc1df88.png">

Reply:
That looks about as nice as one could expect for legacy IE, and it looks like all of our other supported browsers support `grid`, so everything looks good here. Thanks for checking!